ow_sim_tests/test/arm_check_action.py

```diff
@@ -8,7 +8,9 @@
 import rospy
 import roslib
 import unittest
+import ow_lander.constants
 import ow_lander.msg
+import owl_msgs.msg
 import argparse
 
 from action_testing import *
```

Comment:
I cannot comment on the line, but go ahead and remove `DEFAULT_GROUND_HEIGHT  = -0.155` on line 18 as part of this work and replace any instances of `DEFAULT_GROUND_HEIGHT` with `ow_lander.constants.DEFAULT_GROUND_HEIGHT`